components/BaseMap/BaseMapLayer.tsx

```diff
@@ -13,6 +13,31 @@ const BaseMapLayer: FunctionComponent<BaseMapLayerProps> = ({ map, show, ...opti
     }
     if (show) {
       map.addLayer(options);
+      map.on('click', 'highlight', function(e){
+        if(e.features![0].properties){
+          const geometry = e.features![0].geometry;
+          if(geometry.type === 'Polygon'){
+          const coordinates = geometry.coordinates[0][0];
+          new mapboxgl.Popup({
+            offset: 5,
+            closeOnClick: true
+           })
+           .setLngLat([coordinates[0], coordinates[1]])
+           .setHTML('<h3>' + e.features![0].properties.DName2019 +'</h3>')
```

Comment:
You're hardcoding properties that only apply to one style - Uganda. This won't work for a Kenya map.
Also, you don't seem to have your linter setup properly, cos this has a lot of errors ... 

![image](https://user-images.githubusercontent.com/5672438/75683186-b8539280-5ca7-11ea-82d3-eef08f3a76bb.png)